packages/core/http/core-http-server/src/router/route_validator.ts

```diff
@@ -168,11 +168,15 @@ export type RouteValidatorFullConfigRequest<P, Q, B> = RouteValidatorConfig<P, Q
  */
 export interface RouteValidatorFullConfigResponse {
   [statusCode: number]: {
+    /**
+     * A description of the response. This is required input for complete OAS documentation.
+     */
+    description?: string;
     /**
      * A string representing the mime type of the response body.
      */
     bodyContentType?: string;
-    body: LazyValidator;
+    body?: LazyValidator;
```

Comment:
This is what caused the need for a number of changes (similar for versioning router). The possibility of having empty response schemas was overlooked in the initial implementation. You might have, for example:

```
DELETE /api/my-resource
```

That responds with `OK 201` and an empty body. This enables use to represent this in code and OAS.